CHANGELOG.rst

```diff
@@ -11,6 +11,16 @@ Change Log
 
 .. There should always be an "Unreleased" section for changes pending release.
 
+Unreleased
+----------
+
+
+7.2.0 - 2025-02-26
+------------------
+Removed
+~~~~~~~
+* Removed deprecated monitoring middleware Classes
```

Comment:
[inform/request] This really should have been a `BREAKING CHANGE` 8.0.0 release, even though we hope 🤞 all cases were take care of. It is possible that others have services that are relying on this.
- I added a `!` to the PR title.
- The changelog could be updated with `BREAKING CHANGE`. I'll sometimes add a note like: "All known uses were removed from the Open edX platform.", so people know they probably don't need to be concerned.
- Maybe we should add a note to the next named release docs, but maybe that doesn't exist? I see https://openedx.atlassian.net/wiki/spaces/OEPM/pages/4332584961/SUMAC+-+Product+Release+Notes, but I _guess_ this would be for Teak? I'm not sure anymore.